arkouda_viz/client/arkouda_viz/viz.py

```diff
@@ -1,13 +1,28 @@
 """Module providing our plotting capabilities."""
 import holoviews as hv
+from arkouda.pdarrayclass import create_pdarray
 import panel as pn
 import panel.widgets as pnw
+from arkouda.client import generic_msg
 from arkouda.dataframe import DataFrame
 from typeguard import typechecked
 from holoviews.operation.datashader import datashade as ds
+from typing import cast
+import arkouda as ak
 
 pn.extension()
 
+def datashade_server(data: DataFrame, xBins: int = 100, yBins: int = 100):
+    repMsg = generic_msg(cmd="datashade_server",
+                              args={"column_1": data[data.columns[0]],
+                                    "column_2": data[data.columns[1]],
```

Comment:
The assumption would be the user can specify the starting 2 columns as optional parameters but if none are specified, it will default to the first and second columns (if size permits).